.github/CHANGELOG.md

```diff
@@ -2,11 +2,23 @@
 
 <h3>New features since last release</h3>
 
-- Added the `ControlledPhaseShift` gate as well as the `QFT` operation for applying quantum Fourier
-  transforms.
-  [(#1064)](https://github.com/PennyLaneAI/pennylane/pull/1064)
+<h4>JAX is now supported on all devices</h4>
+
+* JAX can now be used as the interface to any device
+
+  Here is an example of how to use JAX with Cirq:
+
+  ```python
+  dev = qml.device('cirq.simulator', wires=1)
+  @qml.qnode(dev, interface="jax")
+  def circuit(x):
+      qml.RX(x[1], wires=0)
+      qml.Rot(x[0], x[1], x[2], wires=0)
+      return qml.expval(qml.PauliZ(0))
+
+  weights = jnp.array([0.2, 0.5, 0.1])
+  print(circuit(weights)) # DeviceArray(...)
```

Comment:
I can't suggest it, but this block is missing an end triple-back-quote

Reply:
https://github.com/PennyLaneAI/pennylane/runs/1855194494#step:4:2022

Reply:
Aw thank you. I was wondering why the doc build kept failing here but worked locally.